src/tape/ensemble.py

```diff
@@ -1032,20 +1032,26 @@ def batch(self, func, *args, meta=None, use_map=True, compute=True, on=None, lab
             the results. Overridden by TAPE for TAPE and `light-curve`
             functions. If none, attempts to coerce the result to a
             pandas.Series.
+        by_band: `boolean`
```

Comment:
Could you please add here a sentence about output column names?

Reply:
I rewrote the sentence that tried to explain how the columns affected, let me know if that  adds any clarity